app/src/main/kotlin/com/wire/android/ui/authentication/welcome/WelcomeScreen.kt

```diff
@@ -206,7 +208,7 @@ private fun WelcomeContent(
                                 )
                             )
                         } else {
-                            navigate(NavigationCommand(CreateTeamAccountOverviewScreenDestination(state)))
+                            CustomTabsHelper.launchUrl(context, teamCreationUrl)
```

Comment:
Shouldn't this be as well behind `ENABLE_NEW_REGISTRATION` feature flag or regarding teams creation we need to immediately replace this for both old and new flow?